### ISSUE TITLE
Add ipv6 support

### ISSUE THREAD

**Issue by anadrianmanrique** [fortra/impacket#1811]
TBD

**Comment by fsacer** [comment]
Most examples do support IPv6, so not sure what exactly is the issue here?

**Comment by anadrianmanrique** [comment]
@fsacer true! but some others not. Some components in the lib don't support ipv6 either.

**Comment by gabrielg5** [comment]
Closing this one as #2023 and #2024 have been merged now
